doc/usage/manual_restoration.md

```diff
@@ -26,7 +26,8 @@ You may choose to follow different methods of restoration, based on your etcd +
         - --embedded-etcd-quota-bytes=<same as previous value>
         - --snapstore-temp-directory=<same as previous value>
         ```
-
+        :warning: Edit etcd-main statefulset to change command field. After saving it, delete etcd-main-0 pod and then check etcd and backup-restore sidecar logs for successful restoration.
```

Comment:
```suggestion
        :warning: Edit etcd-main statefulset to change command field. After saving it, ideally statefulset controller will recreate the etcd-main-0 pod itself. But, since it doesn't handle the case of unhealthy pod,  delete etcd-main-0 pod maually using commnad `kubectl -n <namespace> delete etcd-main-0`. Now, check etcd and backup-restore sidecar logs for successful restoration.
```